notification-hubs-test-app-java/src/main/java/com/example/notification_hubs_test_app_java/MainActivity.java

```diff
@@ -26,5 +26,6 @@ protected void onCreate(Bundle savedInstanceState) {
         InstallationTemplate testTemplate = new InstallationTemplate();
         testTemplate.setBody("{\"data\":{\"message\":\"Notification Hub test notification: $myTextProp\"}}");
         NotificationHub.setTemplate("testTemplate", testTemplate);
+        NotificationHub.setUserId("com.example.notification_hubs_test_app_java");

```

Comment:
[nit] this will show up when we do searches for the package name. Let's change this to something like `john.doe@azure.com`